### ISSUE TITLE
Installation fails on macOS

### ISSUE THREAD

**Issue by JanC** [mapbox/mbview#85]
Hi,
I'm trying to install mbview on macOS Mojave but the npm  installation fails when compiling the sqlite dependency:

Any ideas?
```bash
$ node -v
v11.5.0
$ npm -version
6.6.0
```

<details><summary>Log output</summary>

```
npm install mbview

> sqlite3@3.1.13 install /Users/jan/temp/node_modules/sqlite3
> node-pre-gyp install --fallback-to-build

node-pre-gyp ERR! Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v67-darwin-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for sqlite3@3.1.13 and node@11.5.0 (node-v67 ABI) (falling back to source compile with node-gyp)
node-pre-gyp ERR! Tried to download(undefined): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v67-darwin-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for sqlite3@3.1.13 and node@11.5.0 (node-v67 ABI) (falling back to source compile with node-gyp)
  ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c
  ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c
  TOUCH Release/obj.target/deps/action_before_build.stamp
  TOUCH Release/obj.target/deps/action_before_build.stamp
  CC(target) Release/obj.target/sqlite3/gen/sqlite-autoconf-3150000/sqlite3.o
  CC(target) Release/obj.target/sqlite3/gen/sqlite-autoconf-3150000/sqlite3.o
rm: ./Release/.deps/Release/obj.target/sqlite3/gen/sqlite-autoconf-3150000/sqlite3.o.d.raw: No such file or directory
make: *** [Release/obj.target/sqlite3/gen/sqlite-autoconf-3150000/sqlite3.o] Error 1
gyp  LIBTOOL-STATIC Release/sqlite3.a
 ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:189:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:254:12)
gyp ERR! System Darwin 18.2.0
gyp ERR! command "/usr/local/Cellar/node/11.5.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/Users/jan/temp/node_modules/sqlite3/lib/binding/node-v67-darwin-x64/node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=/Users/jan/temp/node_modules/sqlite3/lib/binding/node-v67-darwin-x64"
gyp ERR! cwd /Users/jan/temp/node_modules/sqlite3
gyp ERR! node -v v11.5.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/Cellar/node/11.5.0/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/jan/temp/node_modules/sqlite3/lib/binding/node-v67-darwin-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/Users/jan/temp/node_modules/sqlite3/lib/binding/node-v67-darwin-x64' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/jan/temp/node_modules/sqlite3/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:189:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:978:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:265:5)
node-pre-gyp ERR! System Darwin 18.2.0
node-pre-gyp ERR! command "/usr/local/Cellar/node/11.5.0/bin/node" "/Users/jan/temp/node_modules/sqlite3/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /Users/jan/temp/node_modules/sqlite3
node-pre-gyp ERR! node -v v11.5.0
node-pre-gyp ERR! node-pre-gyp -v v0.6.38
node-pre-gyp ERR! not ok
Failed to execute '/usr/local/Cellar/node/11.5.0/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/jan/temp/node_modules/sqlite3/lib/binding/node-v67-darwin-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/Users/jan/temp/node_modules/sqlite3/lib/binding/node-v67-darwin-x64' (1)
  CXX(target) Release/obj.target/node_sqlite3/src/database.o3~install: Failed to exec install script
error: error reading '../src/database.cc'cycle sqlite3@3.1.13~install: Failed to exec install script
1 error generated.
make: *** [Release/obj.target/node_sqlite3/src/database.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:189:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:254:12)
gyp ERR! System Darwin 18.2.0
gyp ERR! command "/usr/local/Cellar/node/11.5.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/Users/jan/temp/node_modules/sqlite3/lib/binding/node-v67-darwin-x64/node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=/Users/jan/temp/node_modules/sqlite3/lib/binding/node-v67-darwin-x64"
gyp ERR! cwd /Users/jan/temp/node_modules/sqlite3
gyp ERR! node -v v11.5.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm WARN enoent ENOENT: no such file or directory, open '/Users/jan/temp/package.json'
npm WARN temp No description
npm WARN temp No repository field.
npm WARN temp No README data
npm WARN temp No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sqlite3@3.1.13 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sqlite3@3.1.13 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jan/.npm/_logs/2019-01-21T21_07_38_604Z-debug.log
```
</details>

**Comment by JanC** [comment]
thanks!
The installation works for me now.

**Comment by tcql** [comment]
merged and published as v2.0.1
